tools/cmd/api-generator/metadata.go.tmpl

```diff
@@ -20,27 +20,26 @@ package main
 
 import 	"github.com/mongodb/mongodb-atlas-cli/atlascli/tools/internal/metadatatypes"
 
-var EndpointExamples = map[string]metadatatypes.Metadata{
-{{- range $operationID, $examples := . }}
+var metadata = map[string]metadatatypes.Metadata{
+{{- range $operationID, $metadata := . }}
 	`{{ $operationID }}`: {
-		ParameterExample: map[string]string{
-		{{- range $paramName, $example := $examples.ParameterExample }}
-			`{{ $paramName }}`: `{{ $example }}`,
-		{{- end }}
-		},
-		RequestBodyExamples: map[string][]metadatatypes.RequestBodyExample{
-		{{- range $version, $requestBodyExamples := $examples.RequestBodyExamples }}
-			`{{ $version }}`: {
-				{{- range $example := $requestBodyExamples }}
-				{
-					Name:        `{{ $example.Name }}`,
-					Description: `{{ $example.Description }}`,
-					{{if $example.Value}}Value:       `{{ $example.Value }}`,{{end}}
-				},
-				{{- end }}
-			},
-		{{- end }}
-		},
-	},
-{{- end }}
-}
\ No newline at end of file
+		Parameters: {{if not $metadata.Parameters}}nil,{{else}}map[string]metadatatypes.ParameterMetadata{
+		{{- range $paramName, $paramMetadata := $metadata.Parameters }}
+			`{{ $paramName }}`: metadatatypes.ParameterMetadata{
+				Example: `{{ $paramMetadata.Example | replace "`" "` + \"`\" + `" }}`,
```

Comment:
Would it have been cleaner to create a `escape` function instead?

Reply:
backticks (`) do not allow escape but do allow multiline 🤷